Tests/SonarScanner.MSBuild.PreProcessor.Test/SonarWebServiceTest.cs

```diff
@@ -210,19 +216,17 @@ public void TryGetQualityProfile_MultipleQPForSameLanguage_ShouldThrow()
-            mockDownloader.Setup(x => x.Download(new Uri($"{serverUrl}/api/server/version"), false)).Returns(Task.FromResult("8.0.0.22548"));
-            mockDownloader.Setup(x => x.TryDownloadIfExists(new Uri($"{serverUrl}/api/qualityprofiles/search?project={projectKey}&organization=ThisIsInvalidValue"), false)).Returns(Task.FromResult(Tuple.Create(false, (string)null)));
-            mockDownloader.Setup(x => x.Download(new Uri($"{serverUrl}/api/qualityprofiles/search?defaults=true&organization=ThisIsInvalidValue"), false)).Returns(Task.FromResult<string>(null));    // SC returns 404, WebClientDownloader returns null
+            mockDownloader.Setup(x => x.Download(new Uri($"{ServerUrl}/api/server/version"), false)).Returns(Task.FromResult("8.0.0.22548"));
+            mockDownloader.Setup(x => x.TryDownloadIfExists(new Uri($"{ServerUrl}/api/qualityprofiles/search?project={ProjectKey}&organization=ThisIsInvalidValue"), false)).Returns(Task.FromResult(Tuple.Create(false, (string)null)));
+            mockDownloader.Setup(x => x.Download(new Uri($"{ServerUrl}/api/qualityprofiles/search?defaults=true&organization=ThisIsInvalidValue"), false)).Returns(Task.FromResult<string>(null));    // SC returns 404, WebClientDownloader returns null
             mockDownloader.Setup(x => x.Dispose());
-            using (var service = new SonarWebService(mockDownloader.Object, serverUrl, logger))
+            using (var service = new SonarWebService(mockDownloader.Object, ServerUrl, logger))
             {
-                Action a = () => _ = service.TryGetQualityProfile("projectKey", null, "ThisIsInvalidValue", "cs").Result;
+                Action a = () => _ = service.TryGetQualityProfile(ProjectKey, null, "ThisIsInvalidValue", "cs").Result;
                 a.Should().Throw<AggregateException>().WithMessage("One or more errors occurred.");
-                logger.AssertErrorLogged("Failed to request and parse 'http://localhost:42424/api/qualityprofiles/search?defaults=true&organization=ThisIsInvalidValue': Cannot download quality profile. Check scanner arguments and the reported URL for more information.");
-                logger.AssertErrorLogged("Failed to request and parse 'http://localhost:42424/api/qualityprofiles/search?project=projectKey&organization=ThisIsInvalidValue': Cannot download quality profile. Check scanner arguments and the reported URL for more information.");
+                logger.AssertErrorLogged($"Failed to request and parse '{ServerUrl}/api/qualityprofiles/search?defaults=true&organization=ThisIsInvalidValue': Cannot download quality profile. Check scanner arguments and the reported URL for more information.");
+                logger.AssertErrorLogged($"Failed to request and parse '{ServerUrl}/api/qualityprofiles/search?project={ProjectKey}&organization=ThisIsInvalidValue': Cannot download quality profile. Check scanner arguments and the reported URL for more information.");
```

Comment:
I would prefer to see hardcoded values in assertions. To really see the values as they are.